kotlin/toggle-between-feature-request-modes/README.metadata.json

```diff
@@ -0,0 +1,29 @@
+{
+    "category": "Edit and Manage Data",
+    "description": "Use different feature request modes to populate the map from a service feature table.",
+    "formal_name": "ToggleBetweenFeatureRequestModes",
+    "ignore": false,
+    "images": [
+        "toggle-between-feature-request-modes.png"
+    ],
+    "keywords": [
+        "cache",
+        "data",
+        "feature",
+        "feature request mode",
+        "performance",
+        "FeatureLayer",
+        "ServiceFeatureTable",
+        "ServiceFeatureTable.FeatureRequestMode"
+    ],
+    "language": "kotlin",
```

Comment:
we'll need to add redirect URLs. @vquach2404 can help explain it since she did this for iOS